testing/test_conftest.py

```diff
@@ -161,6 +162,10 @@ def pytest_addoption(parser):
     result = testdir.runpytest("-h", "--confcutdir=%s" % x, x)
     result.stdout.fnmatch_lines(["*--xyz*"])
 
+def test_no_conftest(testdir):
+    testdir.makeconftest("assert 0")
+    testdir.runpytest("-h", "--noconftest")
+
```

Comment:
i think "-h" specifically ignores conftest loading problems IIRC.  Don't do "-h" and you should see an exception.

``res = testdir.runpytest... ; assert res.result == 0``